src/ControlTasks/process_frame.py

```diff
@@ -0,0 +1,139 @@
+# break up into smaller control tasks as needed
+# from ..vision.process_frame_util import process_frame
+from .control_task_base import ControlTaskBase
+
+import numpy as np
+import cv2
+import math
+import time
+
+def get_features(frame):
+
+    orb = cv2.ORB_create()
+
+    # Replacement for orb.detect(frame, None) Gives many more points
+    pts = cv2.goodFeaturesToTrack(np.mean(frame, axis=2).astype(np.uint8), 1000, qualityLevel=0.01, minDistance=7)
+
+    # print("pts: ", alt_pts)
+
+    kps = [cv2.KeyPoint(x=f[0][0], y=f[0][1], size=20) for f in pts]
+    kps, des = orb.compute(frame, kps)
+
+    # print("kps: ", kps)
+
+    # return np.array([(kp.pt[0], kp.pt[1]) for kp in kps]), des
+    return kps, des
+
+
+# def get_features2(frame):
+
+#     orb = cv2.ORB_create()
+
+
+def match_frames(des1, des2):
+    # print(des1, des2)
```

Comment:
I think adding comments here would be useful (i.e what type of methods do you use to match each frame?)

Reply:
^^ seconded